clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/remote/RemoteCacheContainerConfigurationServiceConfigurator.java

```diff
@@ -135,6 +140,11 @@ public ServiceConfigurator configure(OperationContext context, ModelNode model)
 
         this.server = context.hasOptionalCapability(CommonRequirement.MBEAN_SERVER.getName(), null, null) ? new ServiceSupplierDependency<>(CommonRequirement.MBEAN_SERVER.getServiceName(context)) : null;
 
+        this.properties.clear();
+        for (Property property : ModelNodes.optionalPropertyList(Attribute.PROPERTIES.resolveModelAttribute(context, model)).orElse(Collections.emptyList())) {
```

Comment:
N.B. Once we upgrade to jboss-dmr 1.6.x, we can replace this with:
```java
for (Property property : Attribute.PROPERTIES.resolveModelAttribute(context, model).asPropertyListOrEmpty())
```

Reply:
Thanks - noted and put on a list.